src/core/service.c

```diff
@@ -2958,9 +3018,19 @@ static int service_stop(Unit *u) {
 
 static int service_reload(Unit *u) {
         Service *s = ASSERT_PTR(SERVICE(u));
+        int r;
 
         assert(IN_SET(s->state, SERVICE_RUNNING, SERVICE_EXITED));
 
+        /* If we have confexts extensions, try to reload vpick'd confext
+         * extensions, which is particularly beneficial for notify-reload
+         * services that could potentially pick up a new version of its
+         * configuration.
```

Comment:
btw, we typically line break at 109ch

Reply:
Thank you! I reflowed the other comments too.